doc/contents/Tablesaw_display_as_TableDisplay.ipynb

```diff
@@ -8,49 +8,26 @@
     "\n",
     "BeakerX allows you to register methods to display 3rd party classes such as Tablesaw's Tables.\n",
     "You can either write such a method in a cell yourself, or load it from a JAR.\n",
-    "We include such a JAR in the documentation, and this tutorial demonstrates it.\n",
-    "\n",
-    "Download the Tablesaw jar and put it in the same directory as this notebook is in.\n",
-    "(http://central.maven.org/maven2/com/github/lwhite1/tablesaw/0.7.6.4/tablesaw-0.7.6.4.jar)"
+    "We include such a JAR in the documentation, and this tutorial demonstrates it."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Added jar: [tablesaw-0.7.6.4.jar]\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "%classpath add jar tablesaw-0.7.6.4.jar\n",
-    "%import com.github.lwhite1.tablesaw.api.*\n",
-    "%import com.github.lwhite1.tablesaw.api.ml.clustering.*\n",
-    "%import com.github.lwhite1.tablesaw.api.ml.regression.*\n",
-    "%import com.github.lwhite1.tablesaw.columns.*    \n",
-    "%import com.github.lwhite1.tablesaw.reducing.*   "
+    "%classpath add mvn tech.tablesaw tablesaw-plot 0.10.0\n",
+    "%import tech.tablesaw.api.Table"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Added jar: [tablesaw-0.7.6.4_Demo.jar]\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "%classpath add jar demoResources/tablesaw-0.7.6.4_Demo.jar\n",
+    "%classpath add jar demoResources/tablesaw_0.10.0_Demo.jar\n",
```

Comment:
what's in this jar? can we add a pointer to the source code?

Reply:
The src for the jar is in the markdown of the notebook, it's more clearly visible here: https://github.com/twosigma/beakerx/blob/73ac34418eeed06df280b774dc51d44857dd5730/doc/contents/Tablesaw_display_as_TableDisplay.ipynb